apps/prairielearn/src/lib/user.ts

```diff
@@ -1,23 +1,23 @@
 import { HttpStatusError } from '@prairielearn/error';
 
 /**
- * Parses a string of UIDs separated by commas, whitespace, line breaks, or semicolons
- * into an array of unique UIDs.
+ * Parses a string of values separated by commas, whitespace, line breaks, or semicolons
+ * into an array of unique values.
  *
- * @param uidsString The string of UIDs to parse.
- * @param limit The maximum number of UIDs to accept without throwing an error.
+ * @param valuesString The string of values to parse.
+ * @param limit The maximum number of values to accept without throwing an error.
  */
-export function parseUidsString(uidsString: string, limit: number): string[] {
-  const uids = new Set(
-    uidsString
+export function parseUniqueValuesFromString(valuesString: string, limit = 1000): string[] {
```

Comment:
We still want this function on the frontend since it can provide richer error messages than Zod. That being said, we can roll email validation into this function if we want. Currently, functions that validate email format do it afterwards and seperately. See `ExtensionModifyModal`

Reply:
Discussed on call that this function won't do email validation.